packages/p2p-v2/src/pages/my-ads/components/AdWizard/AdWizard.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { TStep } from 'types';
+import { FormProgress, Wizard } from '@/components';
+import { LabelPairedXmarkLgBoldIcon } from '@deriv/quill-icons';
+import { Button, Text, useDevice } from '@deriv-com/ui';
+import { AdProgressBar } from '../AdProgressBar';
+import { AdTypeSection } from '../AdTypeSection';
+import './AdWizard.scss';
+
+type TAdWizardNav = {
+    currency: string;
+    localCurrency?: string;
+    rateType: string;
+    steps: TStep[];
+};
+
+const AdWizard = ({ currency, localCurrency, rateType, steps }: TAdWizardNav) => {
+    const { isMobile } = useDevice();
+    const [currentStep, setCurrentStep] = useState(0);
+
+    return (
+        <Wizard
+            className='p2p-v2-ad-wizard'
+            initialStep={0}
+            nav={
+                <div>
+                    {!isMobile ? (
```

Comment:
nit: since we are not using isMobile anywhere else here, can we use isDesktop here instead of !isMobile

```suggestion
                    {isDesktop ? (
```